### DISCUSSION TITLE
Question on SimpleCalculator sample landscape display on android

### DISCUSSION THREAD

**Discussion by modula2019** [unoplatform/uno#18923] [Q&A] [answered]
Hi,
Please could someone help me here. I downloaded the SimpleCalc sample code here `https://github.com/unoplatform/Uno.Samples/tree/master/reference/SimpleCalc/XAML-MVUX`

I was able to deploy it on android and it renders well in portrait but when I tried to render it in landscape, it does not show all screen.

What can I do to resolve the landscape display? Thanks

Portrait
![WhatsApp Image 2024-11-26 at 12 28 27_4b3e6e49](https://github.com/user-attachments/assets/7a885d7e-628e-438f-b1f3-84481f8924e2)

Landscape
![WhatsApp Image 2024-11-26 at 12 28 59_3feae3a0](https://github.com/user-attachments/assets/f0eaf891-1893-4add-9216-851fe2ab9831)

**Comment by erikvilima** [comment, marked as answer]
Hey @modula2019,

Right now, the SimpleCalc sample is set up for portrait mode only. To get it working in landscape, you can check out the Responsive Extensions docs. They show how to make your layout adapt to different screen sizes and orientations.
 
Here’s the link to the docs: [Responsive Extensions](https://platform.uno/docs/articles/external/uno.toolkit.ui/doc/helpers/responsive-extension.html)

Hope that helps! Let me know if you have any other questions.

**Reply by modula2019** [reply]
@eriklimakc Thanks, I will checkout this
